api-reference/beta/resources/onpremisesprovisioningerror.md

```diff
@@ -0,0 +1,45 @@
+# onPremisesProvisioningError resource type
+
+Represents directory sync errors during provisioning. When a User, Group, Contact contains a collection of onPremisesProvisioningErrors, it often indicates key end-user is not functioning correctly. For example, the user cannot sign in or the user cannot send or receive email.
+
+### Properties
+
+| Property | Type | Description |
+|:---------------|:--------|:----------|
+|category|String| Category of the provisioning error. Note: At this time, there is only one possible value. Possible value: *PropertyConflict* |
+|occurredDateTime|DateTimeOffset| The date and time at which the error occurred. |
+|propertyCausingError|String| Name of the directory property causing the error. Possible values: *UserPrincipalName* or *ProxyAddress* |
```

Comment:
I changed this to "Current possible values:"  We prefer to not explicitly say there is a future change just in case it doesn't happen.